docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/PriorTrainerSample.cs

```diff
@@ -1,11 +1,9 @@
 ﻿using System;
-using System.Linq;
 using Microsoft.ML.Data;
-using Microsoft.ML.Trainers;
 
 namespace Microsoft.ML.Samples.Dynamic
 {
-    public class PriorTrainerSample
+    public class PriorTrainer
```

Comment:
>PriorTrainer [](start = 17, length = 12)

if you decide to update PR for nit comments, can you move this class and RandomTrainer to BinaryClassification folder? (If you decide to do it, don't forget to update catalog comments which looks on this file)